lhotse/kaldi.py

```diff
@@ -15,6 +14,247 @@
     fastcopy,
     is_module_available,
 )
+import contextlib
+
+
+class KaldiFormatterBase:
```

Comment:
It seems like both input and output formatters are context managers and don't have a common interface otherwise; not sure we need a common base class for that.